src/__tests__/main.test.ts

```diff
@@ -345,3 +346,59 @@ test("upsert where clause", async (t) => {
     t.is(res.data.allRoles.edges.length, 1);
   }
 });
+
+test("upsert where clause omit onConflictUpdate", async (t) => {
+  const upsertDirector = async ({
+    projectName = "sales",
+    title = "director",
+    name = "jerry",
+    rank = 1,
+  }: {
+    projectName?: string;
+    title?: string;
+    name?: string;
+    rank?: number;
+  }) => {
+    const query = nanographql(`
+      mutation {
+        upsertRole(ignore: {name: true}, where: {
```

Comment:
I would prefer that this be an array of (valid) field names rather than a map of field names to boolean, so that you can't override a smart tag, but I wasn't able to figure out how to do that. Feel free to propose an alternative